### DISCUSSION TITLE
UDM spontaneous reset to factory settings

### DISCUSSION THREAD

**Discussion by MarcoDominicus** [fabianishere/udm-iptv#71] [Q&A]
Hi,
Last week my UDM has reset its self to factory setting. I was unable to recover and setup the network again from scratch. Today I had the same issue, and again I’m unable to recover and have to setup the network again from scratch.

Can this be an issue with the IPTV configuration or is my hardware failing? 

Best regards,
Marco

**Comment by fabianishere** [comment]
I don't expect this to be related to the IPTV configuration. You could try disabling it if you want to be sure it is not related to that.

**Reply by MarcoDominicus** [reply]
How can i disable the configuration?

**Reply by fabianishere** [reply]
`systemctl disable udm-iptv`
